guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -62,21 +62,34 @@ The above command registers a client for {Project} in {Keycloak}.
 
 . Ensure that the *Access type* setting is set to *Confidential*.
 
-. If you use {Keycloak} version 7.3 or later, navigate to the {Keycloak} web UI, and add an audience mapper and set the following values for the audience mapper:
+. If you use {Keycloak} version 7.3 or later, complete the following steps:
+
+.. Navigate to the {Keycloak} web UI, click *Clients* and click the client that you use with {Project}.
```

Comment:
can we say something like `click on the client registered with {Prohject}` instead of `click the client that you use with {Project}.`?

Reply:
Sure, thank you for this correction. Done.